_gsocproposals/2022/proposal_Geant4-Symplectic-integrators.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Symplectic integrators for Geant4
```

Comment:
What about to prefix the title with the name of the project? I find worthwhile to put together proposals from a project in the full list…